ietf-sip-auto-peering@2025-03-31.yang

```diff
@@ -54,7 +54,7 @@ module ietf-sip-auto-peering {
      described in BCP 14 (RFC 2119) (RFC 8174) when, and only when,
      they appear in all capitals, as shown here.";
 
-  revision 2025-03-19 {
+  revision 2025-03-31 {
```

Comment:
This copy had an incorrect revision so I hope this is the correct source of truth